NEWS.md

```diff
@@ -1,5 +1,8 @@
 # pkgdown (development version)
 
+* If `SUPPORT.md` is present in `/.github` folder, it will be included on
+  homepage as "Getting help" (@patilindrajeets, #2124).
```

Comment:
Isn't your github user name @IndrajeetPatil?

Reply:
LOL. Sorry, included the Twitter handle instead of GitHub! 😅